src/utils/rpc/services/item.RPCservice.ts

```diff
@@ -24,40 +22,12 @@ export class ItemRPCService {
     });
   }
 
-  static async createMissionItem(
-    title: string,
-    contentType: ContentType,
-    priority?: number
-  ): Promise<IItem> {
-    return RPCClientRequest(ItemRPCService.rpcClient, "createMissionItem", {
-      title,
-      contentType,
-      priority,
-    });
-  }
-
   static async createItem(item: IItem): Promise<IItem> {
     return RPCClientRequest(ItemRPCService.rpcClient, "createItem", {
       item,
     });
   }
 
-  static async getAreaById(areaId: string): Promise<IArea> {
```

Comment:
make sure to delete uses of those functions in backend

Reply:
I did. It would be merged w/ a corresponding cr in user-service